.github/workflows/ci.yml

```diff
@@ -0,0 +1,175 @@
+name: EFCore.PG CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        dotnet: [ '3.0.100' ]
+        pg_major: [ 11, 10 ]
+        include:
+          - pg_major: 11
+            edb_version: '11.4-2'
+            postgis_version: '2.5.3'
+          - pg_major: 10
+            edb_version: '10.9-2'
+            postgis_version: '2.5.3'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (docker)
+        if: startsWith(matrix.os, 'ubuntu')
+        env:
+          IMAGE: npgsql/postgres:${{ matrix.pg_major }}
+        run: docker run --detach --name postgresql -p 5432:5432 $IMAGE
+        shell: bash
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (install)
+        if: startsWith(matrix.os, 'windows')
+        env:
+          PG_MAJOR: ${{ matrix.pg_major }}
+          EDB_VERSION: ${{ matrix.edb_version }}
+          POSTGIS_VERSION: ${{ matrix.postgis_version }}
+        run: |
+          # Install PostgreSQL
+          curl -o pgsql.zip -L https://get.enterprisedb.com/postgresql/postgresql-${EDB_VERSION}-windows-x64-binaries.zip
+          unzip pgsql.zip -x 'pgsql/include/**' 'pgsql/doc/**' 'pgsql/pgAdmin 4/**' 'pgsql/StackBuilder/**'
+
+          # Match Npgsql CI Docker image and stash one level up
+          cp {$GITHUB_WORKSPACE/.build/docker,pgsql}/server.crt
+          cp {$GITHUB_WORKSPACE/.build/docker,pgsql}/server.key
+
+          # Install PostGIS
+          POSTGIS_FILE="postgis-bundle-pg${PG_MAJOR}-${POSTGIS_VERSION}x64"
+          curl -o postgis.zip -L https://download.osgeo.org/postgis/windows/pg${PG_MAJOR}/${POSTGIS_FILE}.zip
+          unzip postgis.zip -d postgis
+          cp -a postgis/$POSTGIS_FILE/. pgsql/
+
+          # Start PostgreSQL
+          pgsql/bin/initdb -D pgsql/PGDATA -E UTF8 -U postgres
+          pgsql/bin/pg_ctl -D pgsql/PGDATA -l logfile -o '-c max_prepared_transactions=10 -c ssl=true -c ssl_cert_file=../server.crt -c ssl_key_file=../server.key' start
+
+          # Configure test account
+          pgsql/bin/psql -U postgres -c "CREATE ROLE npgsql_tests SUPERUSER LOGIN PASSWORD 'npgsql_tests'"
+          pgsql/bin/psql -U postgres -c "CREATE DATABASE npgsql_tests OWNER npgsql_tests"
+        shell: bash
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.0.2 # Issues with DOTNET_ROOT
+        with:
+          dotnet-version: ${{ matrix.dotnet }}
+
+      - name: Test (EFCore.PG.Tests)
```

Comment:
Not possible to just run dotnet test at the root?

Reply:
I think we could, the idea here was just to make it easier to isolate failures. I'm fine with either option, do you have a preference?

Reply:
Let's start with unifying it and see how it goes... If it turns out to be difficult to figure out where failures come from we can always split back.